resources/content/config-content/dns/content-home/etc/cattle/dns/answers.json.ftl

```diff
@@ -0,0 +1,20 @@
+{
+    "default": {
+        "recurse": ["8.8.8.8"],
+        "rancher.com.": ["1.2.3.4"]
```

Comment:
Defaulting to google 8.8.8.8 is debatable, but `rancher.com.` definitely shouldn't be here...